src/DoctrineORMModule/Service/ConfigurationFactory.php

```diff
@@ -152,6 +152,10 @@ public function __invoke(ContainerInterface $container, $requestedName, array $o
             $config->setSecondLevelCacheEnabled();
             $config->setSecondLevelCacheConfiguration($cacheConfiguration);
         }
+        
+        if($schemaFilter = $options->getSchemaFilter()) {
```

Comment:
We could, but in my opinion it is easier for everyone if we use the same naming as in the symfony repository. There is the config option named schema_filter and so it translates to set/getSchemaFilter. So it's sort of "compatible"..?

Is your suggestion naming the option filter_schema, or filter_schema_assets_expression? The last one would be a little bit too much?